command/rootchain/initcontracts/rootchain_initcontracts.go

```diff
@@ -0,0 +1,321 @@
+package initcontracts
```

Comment:
Why namespace this file name with `rootchain_`?

Reply:
Just to make sure I've understood correctly:
you are suggesting naming package as `rootchain_initcontracts`, `rootchain_server`, `rootchain_emit` and so on?

Reply:
no, the other way around, naming the files `initcontracts` `server` `emit` 😄

Reply:
Fixed [97f3d78](https://github.com/0xPolygon/polygon-edge/pull/796/commits/97f3d78fdf9f276162c65df6b07b1d855b190e3d)